index.html

```diff
@@ -24,25 +24,183 @@
+              <a href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+        <!-- card 3 -->
+        <div class="card">
+          <div class="card-image-container card-3-image"></div>
+          <div class="card-content">
+            <h3>Tonic</h3>
+            <div class="first-links">
+              <a href="#" class="canopy-style">CANOPY</a>
+              <span class="canopy-image"></span>
+              <a href="#">Back End dev</a>
+              <span class="back-end-image"></span>
+              <a href="#">2015</a>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="second-links">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <a href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+        <!-- card 4 -->
+        <div class="card">
+          <div class="card-image-container card-4-image"></div>
+          <div class="card-content">
+            <h3 class="mt-post">Multi-Post-Stories</h3>
+            <div class="first-links">
+              <a href="#" class="canopy-style">CANOPY</a>
+              <span class="canopy-image"></span>
+              <a href="#">Back End dev</a>
+              <span class="back-end-image"></span>
+              <a href="#">2015</a>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="second-links">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <a href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- about me -->
+      <section class="about-me bg-white">
+        <div class="first-section">
+          <h3>About Myself</h3>
           <p>
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contact me.
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
           </p>
+          <div>
+            <p><a href="#">LET'S CONNECT</a></p>
+          </div>
```

Comment:
- [ ] `LET'S CONNECT` is a subtitle in this case, so it would nice if you use `hx` HTML tag eg: h1, h2, h3... Go ahead and courage!😊